include/xrpl/protocol/Asset.h

```diff
@@ -97,6 +97,13 @@ class Asset
 
     friend constexpr bool
     operator==(Currency const& lhs, Asset const& rhs);
+
+    /** Return true if both asset's issue (Issue or MPTIssue)
+     * are the same and hold the same token (currency or MPTID).
+     * Otherwise return false.
```

Comment:
I'm suggesting a change to the whole string, but wanted to note the original string was one of the rare occasions you'd need the plural possessive "assets'" (apostrophe _after_ the s). You'd need to pluralize "issues" too.

```suggestion
    /** Return true if both assets refer to the same currency (regardless of issuer) or MPT issuance.
     * Otherwise return false.
```